src/io/free_functions.rs

```diff
@@ -3,32 +3,7 @@ use std::io::{BufRead, BufReader, BufWriter, Seek};
 use std::path::Path;
 use std::u32;
 
-#[cfg(feature = "bmp")]
-use crate::codecs::bmp;
-#[cfg(feature = "gif")]
-use crate::codecs::gif;
-#[cfg(feature = "hdr")]
-use crate::codecs::hdr;
-#[cfg(feature = "ico")]
-use crate::codecs::ico;
-#[cfg(feature = "jpeg")]
-use crate::codecs::jpeg;
-#[cfg(feature = "png")]
-use crate::codecs::png;
-#[cfg(feature = "pnm")]
-use crate::codecs::pnm;
-#[cfg(feature = "tga")]
-use crate::codecs::tga;
-#[cfg(feature = "dds")]
-use crate::codecs::dds;
-#[cfg(feature = "tiff")]
-use crate::codecs::tiff;
-#[cfg(feature = "webp")]
-use crate::codecs::webp;
-#[cfg(feature = "farbfeld")]
-use crate::codecs::farbfeld;
-#[cfg(any(feature = "avif-encoder", feature = "avif-decoder"))]
-use crate::codecs::avif;
+use crate::codecs::*;
 
```

Comment:
Simplified this huge list of imports - should yield an exactly equivalent import. Instead of checking for each codec and importing it, import all active codecs